mlx/coverity/coverity.py

```diff
@@ -46,7 +46,7 @@ def initialize_environment(self, app):
     \\makeatother"""
 
         self.stream = app.config.coverity_credentials["stream"]
-
+        self.snaphsot = app.config.coverity_credentials["snapshot"]
```

Comment:
```suggestion
        self.snaphsot = app.config.coverity_credentials.get("snapshot", "")
```
This should be optional for backwards compatibility to prevent sphinx-build crashing with `Handler <bound method SphinxCoverityConnector.initialize_environment of <mlx.coverity.coverity.SphinxCoverityConnector object at 0x7a45878753d0>> for event 'builder-inited' threw an exception (exception: 'snapshot')`

Reply:
Resolved in cc38b898d11b701f23e9f9577927fc56b1f1fa78